cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1317,11 +1317,37 @@ declare namespace Deno {
    */
   export function openKv(path?: string): Promise<Deno.Kv>;
 
+  /** **UNSTABLE**: New CronScheduleExpression is used as the type of `minute`,
```

Comment:
This line needs to just have `**UNSTABLE**: New API, yet to be vetted.` The description goes below after a blank line.

Reply:
Done.